src/web/src/components/Posts/PostInfo.tsx

```diff
@@ -9,7 +9,12 @@ const useStyles = makeStyles((theme: Theme) =>
       display: 'flex',
       flexDirection: 'column',
       justifyContent: 'center',
-      borderLeft: '2.5px solid #707070',
```

Comment:
Keep it.

Reply:
@PedroFonsecaDEV  looks weird with this line

Reply:
@PedroFonsecaDEV or keep it but with a very light color

Reply:
@huynguyez we can think about it in the future. For now let's keep ` borderLeft: '2.5px solid #707070'`

Reply:
@huynguyez Keep this line.